src/date.js

```diff
@@ -62,9 +80,12 @@ Globalize.prototype.dateFormatter = function( options ) {
 
 	validateDefaultLocale( cldr );
 
+	timeZone = options.timeZone;
+	validateParameterTypeString( options.timeZone, "options.timeZone" );
```

Comment:
Now that we the additional option `options.timeZone`, we need to update the above `options = options || { skeleton: "yMd" }`. For example, `dateFormatter({timeZone: "something"})` should also set the default `skeleton: "yMd"`. This default doesn't need to be set if either one is used: skeleton, date, or datetime.